astropy/time/core.py

```diff
@@ -2250,8 +2250,8 @@ def light_travel_time(
         )
 
         # Move X,Y,Z to last dimension, to enable possible broadcasting below.
-        cpos = np.rollaxis(cpos, 0, cpos.ndim)
-        spos = np.rollaxis(spos, 0, spos.ndim)
+        cpos = np.moveaxis(cpos, 0, cpos.ndim - 1)
+        spos = np.moveaxis(spos, 0, spos.ndim - 1)
```

Comment:
I think this should be `np.moveaxis(cpos, 0, -1)`.

Reply:
Looking at the table from https://numpy.org/doc/stable/reference/generated/numpy.rollaxis.html the two should be equivalent, but I'd agree `-1` would be clearer to read.

Reply:
I'll push changes later tonight for cleaner format.